src/app/create-question-page/create-question-page.component.ts

```diff
@@ -1,15 +1,143 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { HttpResponse } from '@angular/common/http';
+import { FormsModule } from '@angular/forms';
+import { MatInputModule } from '@angular/material/input';
+import { EditorModule } from '@tinymce/tinymce-angular';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatSnackBar } from '@angular/material';
+
+import { ApiService } from './../_services/api.service';
 
 @Component({
-  selector: 'app-create-question-page',
-  templateUrl: './create-question-page.component.html',
-  styleUrls: ['./create-question-page.component.css']
+    selector: 'app-create-question-page',
```

Comment:
Again, formatting change.. should be in its own commit.

Reply:
Understood and hopefully can find and undo these (spaces and adjustments)  and then re-apply them within the code procedures you recommend. A learnt moment. Thanks!